releasing/kubernetes.md

```diff
@@ -0,0 +1,37 @@
+# Kubernetes
+
+## Releasing to non-production
+
+1. Wait for [the Docker build to complete on CircleCI](https://circleci.com/gh/ministryofjustice/<repo_name>) for the feature branch.
```

Comment:
Might be worth making this not a link now it's just a url template and not actually a particular url to anything

I've pointed out other occurrences of this below, so that whichever way we decide to go we can be consistent

Reply:
@exonian Do you think it's worth removing link and just have the url

Or listing the actual links for the applications
Or as you said, adding a footnote saying the links are just url templates

Reply:
@said-moj Given we don't have super many, the listed links could be nice. I had just been thinking unlink, but I do like that idea

Reply:
@said-moj Actually I think, while it's not all that many repos, it's a lot of places in this readme. I would unlink